mage_ai/frontend/components/CodeBlock/index.tsx

```diff
@@ -139,8 +140,12 @@ function CodeBlockProps({
   const [runEndTime, setRunEndTime] = useState<number>(null);
   const [runStartTime, setRunStartTime] = useState<number>(null);
 
-  const blockMenuRef = useRef(null);
+  useEffect(() => {
+    setCodeCollapsed(get(`${pipeline.uuid}/${block.uuid}/codeCollapsed`, false));
```

Comment:
`${pipeline.uuid}/${block.uuid}/codeCollapsed` and `${pipeline.uuid}/${block.uuid}/outputCollapsed` are repeated a lot. What if we use a simple util method to create the key names or at least create a variable in this file that can be reused?